spec/unit/shoulda/matchers/active_model/validate_presence_of_matcher_spec.rb

```diff
@@ -204,6 +204,89 @@ def model_creator
     end
   end
 
+  # validate_presence_of an attachment only works on rails >= 6
```

Comment:
Instead of adding another version qualifier to RailsVersions (which really should go away), what if you were to add a condition to [ActiveRecordVersions](https://github.com/thoughtbot/shoulda-matchers/blob/master/spec/support/unit/helpers/active_record_versions.rb)? That should allow you to be more expressive here (and you wouldn't need the comment anymore).

Reply:
I'm creating a condition on `ActiveStorageVersions` class. then the helper is `active_storage_supports_validate_presence?`, which, IMO, makes more readable

Reply:
Ah, I'd forgotten that ActiveStorage was added in Rails 5.2. In that case, is it really the case that the presence validation didn't work on `has_one_attached` or `has_many_attached` before Rails 6?

Reply:
Yeah. That's how I noticed the issue in the first place. I had `validate_presence_of` commented since I migrated to ActiveStorage. After upgrading to Rails 6, I uncommented and validation started to work. Then I uncommented the test and it was passing. After pushing to latest version of shoulda gem, I noticed this issue. You can see that it fails with rails 5 here https://travis-ci.com/gmmcal/shoulda-error/jobs/208187338

I never tracked the solution on master Rails repo, but somehow it was fixed.

Reply:
I just noticed that I have to roll this back to `ActiveRecordVersions` because tests fails on rails < 5.2 due to non-existing `ActiveStorage::VERSION::STRING`. I'll stick to your suggestion and not create `ActiveStorageVersions` class